cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -3,23 +3,38 @@
     :copyright: Copyright © 2014-2015 Cask Data, Inc.
 
 ============================================
-Runtime Arguments
+Preferences
 ============================================
 
 .. include:: ../../../_common/_include/include-v260-deprecate-procedures.rst
 
-Flows, Procedures, MapReduce programs, and Workflows can receive runtime arguments:
+Preferences provide the ability to save configuration at various levels of the system including CDAP Instance,
```

Comment:
Preferences provide the ability to save configuration information at various levels of the system, including the CDAP Instance, Namespace, Application, and Program levels. A configuration is represented by a map of string-string pairs. Preferences can be retrieved, saved, and deleted through a RESTful API and through the Command Line Interface. When programs are started, all the Preferences at the different levels are collapsed into a single map. Preferences
are persisted across a restart of Programs and CDAP.

\[**I'm wondering if this description is accurate. Do they persist across a restart of CDAP? Is the collapse when a program is started? Is the collapse just for the preferences related to that program? For instance, CDAP + Namespace + Application at the Application level, and + Program for each program. That way, Program A gets a different map than Program B.**]